### PR TITLE
first notice UI

### PR DESCRIPTION
**Pull request by YuliaSimanavets**

### CODE REVIEW

File: RingSizer/Modules/Ring/View/NoticeAlertViewController.swift [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    private let okButton = OrangeCustomButton(colors: [UIColor.orangeGradientTop.cgColor, UIColor.orangeGradientBottom.cgColor])
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        createBlur()
+        setupContainerView()
+    }
+
+    func setNotice(data: NoticeAlertViewModel) {
```
Comment (vkostechko-applace) [inline RingSizer/Modules/Ring/View/NoticeAlertViewController.swift:48]: avoid set as prefix, it's not common approach in swift 
better setup or something similar

File: RingSizer/Modules/Ring/View/NoticeAlertViewController.swift [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        titleLabel.text = data.title
+        subtitleLabel.text = data.subtitle
+        okButton.setTitle(data.buttonTitle, for: .normal)
+    }
+
+    @objc
+    func okTapped() {
```
Comment (vkostechko-applace) [inline RingSizer/Modules/Ring/View/NoticeAlertViewController.swift:55]: private?

File: RingSizer/Modules/Ring/View/RingViewController.swift [hunk 3]
```diff
@@ -45,17 +51,33 @@ final class RingViewController: UIViewController {
...
-                                                                           mmSizeTitle: L10n.RingScreen.SizeType.mm,
-                                                                           usSizeTitle: L10n.RingScreen.SizeType.us))
+        ringMeasurementCustomView.set(data: .init(euSizeTitle: L10n.RingScreen.SizeType.eu,
+                                                  mmSizeTitle: L10n.RingScreen.SizeType.mm,
+                                                  usSizeTitle: L10n.RingScreen.SizeType.us))
         saveButton.setTitle(L10n.Buttons.save, for: .normal)
     }
 
+    @objc
+    func openHelp() {
```
Comment (vkostechko-applace) [inline RingSizer/Modules/Ring/View/RingViewController.swift:69]: private?

File: RingSizer/Modules/Ring/View/NoticeAlertViewController.swift (hunk 1 above)
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/NoticeAlertViewController.swift]: got it and fixed it

File: RingSizer/Modules/Ring/View/NoticeAlertViewController.swift (hunk 2 above)
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/NoticeAlertViewController.swift]: fixed it

File: RingSizer/Modules/Ring/View/RingViewController.swift [hunk 4]
```diff
@@ -45,17 +51,33 @@ final class RingViewController: UIViewController {
...
+        sendButton.addTarget(self, action: #selector(openSendMenu), for: .touchUpInside)
     }
 
 #warning("TODO: - replace by sending data from the viewModel ")
     func setupTitles() {
         titleLabel.text = L10n.RingScreen.title
-        ringMeasurementCustomView.set(data: RingMeasurementCustomViewModel(euSizeTitle: L10n.RingScreen.SizeType.eu,
-                                                                           mmSizeTitle: L10n.RingScreen.SizeType.mm,
-                                                                           usSizeTitle: L10n.RingScreen.SizeType.us))
+        ringMeasurementCustomView.set(data: .init(euSizeTitle: L10n.RingScreen.SizeType.eu,
```
[the remark this answers is not in the document]
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/RingViewController.swift]: very useful remark, thanks
fixed it

File: RingSizer/Modules/Ring/View/RingViewController.swift [hunk 5]
```diff
@@ -45,17 +51,33 @@ final class RingViewController: UIViewController {
...
         saveButton.setTitle(L10n.Buttons.save, for: .normal)
     }
 
+    @objc
+    func openHelp() {
+    }
+
+#warning("TODO: - change sendModel, and transfer logic to viewModel")
+    @objc
+    func openSendMenu() {
```
[the remark this answers is not in the document]
Comment (YuliaSimanavets) [reply RingSizer/Modules/Ring/View/RingViewController.swift]: understand, fixed it :)

**Review by vkostechko-applace — APPROVED** [review]

**Pull request merged**